library/tools/smt2/default.lean

```diff
@@ -0,0 +1,37 @@
+import system.io
+import system.process
+import .solvers.z3
+import .syntax
+import .builder
+
+meta def smt2 [io.interface] (build : smt2.builder unit) : io string :=
+do z3 ← z3_instance.start,
+   io.put_str (to_string $ to_fmt build),
+   res ← z3^.raw (to_string $ to_fmt build),
+   io.put_str res,
+   return res
+
+open tactic
+open smt2
+open smt2.builder
+
+meta def reflect_prop (e : expr) : tactic (builder unit) :=
+do p ← is_prop e,
+   if p
+   then do
+       tactic.trace e,
+       n ← mk_fresh_name,
```

Comment:
We should need the `tactic.` prefix here since you opened `tactic` namespace above